test/framework/easyconfigs/yeb/Python-2.7.10-ictce-4.1.13.yeb

```diff
@@ -31,8 +31,7 @@ dependencies: [
 #    nice to have an up to date openssl for security reasons
 ]
 
-# fixme: tuple values; see issue #1442
-# osdependencies: [[openssl-devel, libssl-dev, libopenssl-devel]]
+osdependencies: [[openssl-devel, libssl-dev, libopenssl-devel]]
```

Comment:
add a string value too (e.g. `zlib`), + a comment to make it clear this is not an actual `osdependencies` value